frontend/app/src/locales/en.json

```
@@ -134,6 +147,50 @@
     "hint": "Select as the main currency",
     "profit_currency": "Profit Currency"
   },
+  "underlying_token_manager": {
+    "delete_tooltip": "Delete the underlying token",
+    "edit_tooltip": "Edit an underlying token",
+    "hint": "The percentage of an underlying asset in the token. e.g. A uniswap lp token is consisted of two underlying tokens with a 50% weight each.",
+    "labels": {
+      "address": "Address",
+      "tokens": "Underlying tokens",
+      "weight": "Weight"
+    },
+    "tokens": {
+      "address": "Address",
+      "weight": "Weight",
+      "weight_percentage": "{weight} %"
+    },
+    "validation":  {
+      "non_empty": "Please set a weight value",
+      "address_non_empty": "Address cannot be empty",
+      "not_valid": "The specified weight does not seem like a valid value",
+      "out_of_range": "Weight can be from 1-100%"
+    }
+  },
+  "asset_form": {
+    "labels": {
+      "address": "Address",
+      "asset_type": "Asset Type",
+      "coingecko": "Coingecko",
+      "coingecko_hint": "The identifier of the asset on Coingecko. If not set rotki will not be able to retrieve prices for the asset from Coingecko.",
+      "cryptocompare": "CryptoCompare",
+      "cryptocompare_hint": "The identifier of the asset on CryptoCompare. If not set rotki will not be able to retrieve prices for the asset from CryptoCompare",
```

```suggestion
      "cryptocompare_hint": "The identifier of the asset on CryptoCompare. If not given, the symbol is assumed as default. If that does not work and if not otherwise set rotki will not be able to retrieve prices for the asset from CryptoCompare",
```

I wanted to make it a bit clearer. Is it too long?